.changeset/config.json

```
@@ -0,0 +1,12 @@
+{
+  "$schema": "https://unpkg.com/@changesets/config@1.3.0/schema.json",
+  "changelog": [
+    "@changesets/changelog-github",
+    { "repo": "react-hook-form/react-hook-form" }
```

I'm not sure but I think it should be `react-hook-form/resolvers` ?

```suggestion
    { "repo": "react-hook-form/resolvers" }
```

---

oops, copy-paste failure :(